pkg/devfile/parser/data/v2/components_test.go

```diff
@@ -213,7 +249,7 @@ func TestGetDevfileContainerComponents(t *testing.T) {
 				},
 			}
 
-			devfileComponents := d.GetDevfileContainerComponents()
+			devfileComponents, _ := d.GetDevfileContainerComponents(tt.filterOptions)
```

Comment:
same as above, the error should not be ignored. should check unexpected error

Reply:
same as above

Reply:
updated